inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleTableSourceFactory.java

```diff
@@ -95,6 +95,12 @@ public class OracleTableSourceFactory implements DynamicTableSourceFactory {
                             "Optional startup mode for Oracle CDC consumer, valid enumerations are "
                                     + "\"initial\", \"latest-offset\"");
 
+    public static final ConfigOption<Boolean> MIGRATE_ALL =
```

Comment:
Maybe it is can also extracted to class Constants of the mode 'base'?

Reply:
done.